.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+name: Lint
+
+on: pull_request
+
+jobs:
+  lint-check:
+    name: Check for ESLint + Prettier Violations
+    runs-on: ubuntu-latest
```

Comment:
Do we want `ubuntu-latest` over `ubuntu-18.04`?

Reply:
`ubuntu-latest` is 20, not 18 actually. I changed it here since the environment of the linting probably did not matter as much as the actual unit test suite.

Do you have a preference between either? The reason I made this change for mac/ubuntu is cuz the actions gave me a warning when I was running not the latest version when I was shoring up the tests there